features/input.yml

```diff
@@ -1,8 +1,11 @@
-draft_date: 2024-06-05
 name: <input>
-description: The `<input>` element TODO.
+description: The `<input>` element allows the user to input information to a form; a number of types of input data and control widgets are available, depending on the device and browser.
 spec: https://html.spec.whatwg.org/multipage/input.html#the-input-element
-group: forms
+group:
+  - forms
+  - html-elements
+status:
+  compute_from: html.elements.input
 compat_features:
   - api.HTMLInputElement
```

Comment:
Sorry, it's hard to annotate this in the diff, but some of this API need to be split out into their respective features, such as `step`, `stepUp`, `stepDown` (number, since I think that's the oldest of the bunch that use this) or `valueAsDate` (input-date-time). Could you have a look through each `api` item, to double check this list?